docs: expand events documentation (#2166)
ory/docs: docs/actions/live-events.mdx

your Ory Network project, to your own infrastructure. Pipe those events into your own data warehouse, data lake, or flavor of
choice, and use them to power your own analytics, dashboards, data science, and more.

Live event streams are available for Ory Network enterprise contracts. Talk to your account manager or
[reach out directly](https://www.ory.sh/contact/).

## Event shape

Ory emits events for many different actions. All events abide by the same schema:

```json
{
  "name": string,       // The unique name of the event (e.g., "IdentityCreated").
  "version": string,     // The numerical schema version of the event as a string.
  "timestamp": string,  // ISO 8601 formatted timestamp when the event occurred.
  "projectId": string,  // The ID of the Ory project the event belongs to.
  "eventAttributes": map<string, string> // Additional event-specific attributes.
}
```

Example:

```json
{
  "name": "WebhookSucceeded",
  "version": "1",
  "timestamp": "2025-05-16 11:18:56",
  "eventAttributes": {
    "GeoLocationCity": "Munich",
    "WebhookID": "",
    "WebhookTriggerID": "18093e08-0f72-445f-aacb-151fc777a232"
  }
}
```

> **Important:**

Reviewer: ```suggestion
```

please avoid bolding in general except if you talk about UI elements. 
if you want to put extra emphasis on this section you can put it in an h4 (preferred) or an admonition (e.g. :::caution - but use this sparingly please)

- Per the JSON specification, ordering of keys in objects is not guaranteed and should not be relied upon.
- New keys can appear at any time, consumers should be designed to handle this.

> - The exact content in the `eventAttributes` field is currently not documented and not considered stable. That means that no
>   backwards compability guarantees are made at the moment for them.

Reviewer: ```suggestion
- The exact content in the `eventAttributes` field is not documented and not considered stable. That means that no
  backwards compatibility guarantees are made for them.
```

avoid weasel words

> If you intend to write a consumer that needs a detailed schema and stability guarantees about the `eventAttributes` field, please
> [reach out](https://www.ory.sh/contact/).

Reviewer: ```suggestion
If you intend to write a consumer that needs a detailed schema and stability guarantees about the `eventAttributes` field, please
[reach out](https://www.ory.sh/contact/).
```

We should use /contact only for new users that do not have a business relatiponship with Ory. 

If they are already customers - and to use event streaming they need to be an Enterprise customer so we are definitely aware of them - they should open a support ticket, reach out to their account manager or contact support@ory.sh
```suggestion
If you intend to write a consumer that needs a detailed schema and stability guarantees about the `eventAttributes` field, please
[open a support ticket](https://console.ory.sh/support/).
```

## Event names

The following is a list of all events that are currently supported. The first column corresponds to the `name` field in an event.

> New event names may be added in the future, in which case this list will be updated accordingly.

Reviewer: ```suggestion
```

should be obvious, not sure if we need to state this here, I leave it up to you :)

| **PermissionsExpanded**   | Permissions have been expanded, likely increasing access or privileges for certain users or entities. |
| **PermissionsChecked**    | A permission check has been performed to verify if access is allowed for a given action or resource.  |

## Delivery guarantees

> 1. **Possible duplicates:** There might be duplicates in the event stream. While rare (typically caused by retries because of
>    network issues), consumers should be prepared to deal with them, for example by implementing deduplication logic.
> 1. **Best-effort delivery:** Event delivery is best-effort. While most events are delivered reliably, events may be lost under
>    adverse conditions (for example, system crashes during request processing). Delivery is not guaranteed to be "at-least-once".
> 1. **No delivery latency SLA:** We do not offer a Service Level Agreement (SLA) for event delivery duration (the time between an
>    event occurring and it appearing in your consumer or the Ory Console UI). However, typical delivery times are observed to be
>    less than 5 seconds.
> 1. **No ordering guarantees:** Events may arrive in any order, both within a request and across different requests.

Reviewer: ```suggestion
1. Possible duplicates: There might be duplicates in the event stream. While rare (typically caused by retries because of
   network issues), consumers should be prepared to deal with them, for example by implementing deduplication logic.
1. Best-effort delivery: Event delivery is best-effort. While most events are delivered reliably, events may be lost under
   adverse conditions (for example, system crashes during request processing). Delivery is not guaranteed to be "at-least-once".
1. No delivery latency SLA: Ory does not offer a Service Level Agreement (SLA) for event delivery duration (the time between an
   event occurring and it appearing in your consumer or the Ory Console UI). However, typical delivery times are observed to be
   less than 5 seconds.
1. No ordering guarantees: Events may arrive in any order, both within a request and across different requests.
```

see above avoid bolding. 

instead of using "we"/"us" use "Ory" as entity. 

>However, typical delivery times are observed to be less than 5 seconds.

Not sure if we should include this - customers will take this as ironclad promise and complain if it takes longer. I leave the decision to omit/include up to you as you have to deal with complaints if they come in 😉

> If stronger guarantees are required for your use case, please [reach out](https://www.ory.sh/contact/).

Reviewer: ```suggestion
If stronger guarantees are required for your use case, please [open a support ticket](https://console.ory.sh/support/).
```
